control-plane/typeSpecConverter.cpp

```diff
@@ -242,6 +292,26 @@ bool TypeSpecConverter::preorder(const IR::Type_Enum* type) {
     return false;
 }
 
+bool TypeSpecConverter::preorder(const IR::Type_SerEnum* type) {
+    if (p4RtTypeInfo) {
+        auto name = std::string(type->controlPlaneName());
+        auto enums = p4RtTypeInfo->mutable_serializable_enums();
+        if (enums->find(name) == enums->end()) {
+            auto enumTypeSpec = new p4configv1::P4SerializableEnumTypeSpec();
+            auto bitTypeSpec = new p4configv1::P4BitTypeSpec();
+            enumTypeSpec->set_allocated_underlying_type(bitTypeSpec);
```

Comment:
that's a very weird way to do this
why not `enumTypeSpec->mutable_underlying_type()`? you are not using `bitTypeSpec` for anything else.

Reply:
Sorry, I don't understand fully.  'enumTypeSpec->mutable_underlying_type()` returns a `P4BitTypeSpec*`.  Then what do we do?